### ISSUE TITLE
Fast node not syncing with geth v1.3.4 and bnb48 fast snapshot

### ISSUE THREAD

**Issue by tajelp** [bnb-chain/bsc#2032]
#### System information

Geth version:
```
Geth
Version: 1.3.4
Git Commit: 885de2c1ca4c8cd73d7c8c6e6dba479015fb585e
Git Commit Date: 20231204
Architecture: amd64
Go Version: go1.20.11
Operating System: linux
GOPATH=
GOROOT=/opt/hostedtoolcache/go/1.20.11/x64
```

OS & Version: `Linux Ubuntu 22.04.3 LTS`

I downloaded and extracted the binaries and mainnet.zip.
Then I downloaded the bnb48 snapshot from [here](https://github.com/48Club/bsc-snapshots#geth-fast-node), extracted it and moved the `geth` folder inside the bsc datadir. Then I ran the command
```
./geth_linux \
  --datadir ~/.bsc \
  --history.transactions=0 \
  --syncmode=full \
  --tries-verify-mode=none \
  --pruneancient=true \
  --db.engine=pebble \
  --http.port 8565 \
  --cache 10000 \
  --maxpeers 200 \
  --port 30304 \
  --ws \
  --ws.origins "127.0.0.1" \
  --ws.port 8566
```

The problem is that it never synces and, in fact, if I do `eth.syncing` from the `geth attach` shell, it always says `false`.
Here's the geth log: [geth_bsc_log.txt](https://github.com/bnb-chain/bsc/files/13553811/geth_bsc_log.txt)

**Comment by weiihann** [comment]
You are missing the `--config` flag, then the node will default to discovering peers in the ETH mainnet instead of BSC mainnet, that's why the node will never sync. There's a PR already to change the default network from ETH mainnet to BSC, awaiting reviews.

TLDR, add `--config config.toml` to your start up command.
